### PR TITLE
MGMT-12454: get sos report from ARM machine

### PR DESCRIPTION
**Pull request by adriengentil**
- default inventory file if it is not already existing, we expect at
  least one machine in the `primary` group
- reorder the gather playbook to gather common information (sosreport)
  from all the hosts, and specific test-infra information from the
  primary group

### CODE REVIEW

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

File: ci-operator/step-registry/assisted/common/gather/assisted-common-gather-commands.sh [hunk 1]
```diff
@@ -20,32 +20,38 @@ cat > gather_logs.yaml <<-EOF
...
+
+      - name: Gather sosreport from all hosts
+        ansible.builtin.command: >-
+          sos report --batch --tmp-dir "{{ LOGS_DIR }}" --all-logs
+            -o memory,container_log,filesys,kvm,libvirt,logs,networkmanager,networking,podman,processor,rpm,sar,virsh,yum
+            -k podman.all -k podman.logs
+
+      - name: List swtpm-localca files to a file
+        ansible.builtin.shell: >-
+          ls -ltr /var/lib/swtpm-localca/ >> {{ LOGS_DIR }}/{{ ansible_host }}/libvirt-qemu/ls-swtpm-localca-{{ ansible_host }}.txt
```
Comment (adriengentil) [inline ci-operator/step-registry/assisted/common/gather/assisted-common-gather-commands.sh:39]: no, sure if we need to keep that, it is useful?

File: ci-operator/step-registry/assisted/common/gather/assisted-common-gather-commands.sh [hunk 2]
```diff
@@ -20,32 +20,38 @@ cat > gather_logs.yaml <<-EOF
...
-          source /root/config.sh
-          # Get sosreport including sar data
-          sos report --batch --tmp-dir {{ LOGS_DIR }} --all-logs \
-            -o memory,container_log,filesys,kvm,libvirt,logs,networkmanager,networking,podman,processor,rpm,sar,virsh,yum \
-            -k podman.all -k podman.logs
-      - name: Copy libvirt qemu log
-        copy:
-          remote_src: true
-          src: /var/log/swtpm/libvirt/qemu/
-          dest: "{{ LOGS_DIR }}/libvirt-qemu"
```
Comment (adriengentil) [inline ci-operator/step-registry/assisted/common/gather/assisted-common-gather-commands.sh:41]: removed that, it should be included in the sosreport

File: ci-operator/step-registry/assisted/common/gather/assisted-common-gather-commands.sh (hunk 1 above)
Comment (osherdp) [reply ci-operator/step-registry/assisted/common/gather/assisted-common-gather-commands.sh:39]: not anymore

**Review by osherdp — APPROVED** [review]

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

**Review by osherdp — APPROVED** [review]

**Pull request merged**